### ISSUE TITLE
`prefer-to-be` to report `.toMatch()`

### ISSUE THREAD

**Issue by zaicevas** [jest-community/eslint-plugin-jest#1021]
Currently, the following code snippet is NOT reported by `prefer-to-be`:

```
expect(foo).toMatch('bar')
```

Since `.toMatch` takes either a regexp or a string, I think `prefer-to-be` could find out whether `.toMatch()` is called with a string and suggest to use `.toBe` instead. If `.toMatch()` is called with a regexp, then `prefer-to-be` wouldn't report.
